src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -39,7 +55,13 @@
      * @return map {user email : user}
      */
     public Map<String, User> groupByEmail(final List<User> users) {
-        return null;
+
+
+        Map<String,User> usersMap=users.stream()
+                                  .filter(User -> User.getEmail()!=null)
```

Comment:
Lambda parameter is still parameter. According to java conventions we don't use Capitalized words for parameters.